### PR TITLE
Bottom Navbar and add the file structure to readme

### PR DESCRIPTION
**Pull request by ishak52**

### CODE REVIEW

**Comment by ishak52** [comment]
@ashatat @emanKhaledh @ahmadmshatat @MahmoudMH

File: README.md [hunk 1]
```diff
@@ -28,6 +28,54 @@ the App database structure for represent the App tables and the relations betwee
...
+  |       |  |      - Discussions/
+  |       |  |      - Filter/
+  |       |  |      - Header/
+  |       |  |      - MyCarePlan/
+  |       |  |      - MyDiary/
+  |       |  |      - Navbar/
+  |       |  |      - Profile/
+  |       |  |      - Search/
+  |       |  |      - Error/
+  |       |  - AppRoutes.CSS
```
Comment (MahmoudMH) [inline README.md]: this is CSS file for the index.js which is the main file right ?

so please rename it to be index.css

**Comment by yahyaHB** [comment]
Fix conflicts @ishak52

**Comment by ishak52** [comment]
@yahyaHB done

**Pull request merged**